packages/x-charts-pro/src/FunnelChart/index.ts

```diff
@@ -1,4 +1,5 @@
 export { FunnelChart as Unstable_FunnelChart } from './FunnelChart';
+export { FunnelChart } from './FunnelChart';
```

Comment:
Should we do something like this so people are inclined to migrate?

```suggestion
/** 
 * @deprecated funnel chart is now stable, import `FunnelChart` instead
 */
export const Unstable_FunnelChart = FunnelChart;
```